sql-plugin/src/main/31xdb/scala/com/nvidia/spark/rapids/shims/v2/Spark31XdbShimsBase.scala

```diff
@@ -63,7 +63,7 @@ trait Spark31XdbShimsBase extends SparkShims {
   override def newBroadcastQueryStageExec(
       old: BroadcastQueryStageExec,
       newPlan: SparkPlan): BroadcastQueryStageExec =
-    BroadcastQueryStageExec(old.id, newPlan, old.originalPlan)
+    old.newReuseInstance(old.id, newPlan.output).asInstanceOf[BroadcastQueryStageExec]
```

Comment:
same here, this doesn't seem related to shim work?

Reply:
same comment as above